### PR TITLE
Update hololens-cortana.md

### PR DESCRIPTION
**Pull request by tmlyon**
Added system voice commands and Cortana functionality that's new for Vb.

### CODE REVIEW

**Comment by tmlyon** [comment]
Fyi @Teresa-Motiv @TinaMcN, this will need to be handled specially by localization.

**Comment by Teresa-Motiv** [comment]
@denisebmsft, @scooley , since I was assigned this item instead of requested as a reviewer, what is my role here?

File: devices/hololens/hololens-cortana.md [hunk 1]
```diff
@@ -87,31 +99,36 @@ Sometimes it's helpful to spell out things like email addresses. For instance, t
 
 ## Do more with Cortana
 
-Cortana can help you do all kinds of things on your HoloLens, from searching the web to shutting down your device. She can give you suggestions, ideas, reminders, alerts, and more. To get her attention, select Cortana  on **Start** or say "Hey Cortana" anytime.
+Cortana can help you do all kinds of things on your HoloLens, but depending on which version of Windows Holographic you're using, the capablities may be different. You can learn more about the updated capabilites of the latest version of Cortana [here](https://blogs.windows.com/windowsexperience/2020/02/28/cortana-in-the-upcoming-windows-10-release-focused-on-your-productivity-with-enhanced-security-and-privacy/). 
 
 ![Hey Cortana!](images/cortana-on-hololens.png)
 
-Here are some things you can try saying (remember to say "Hey Cortana" first).
+Here are some things you can try saying (remember to say "Hey Cortana" first).
```
Comment (illfated) [inline devices/hololens/hololens-cortana.md:106]: Redundant whitespace at the end of the line:
```suggestion
Here are some things you can try saying (remember to say "Hey Cortana" first).
```
(please remove this whitespace, either by committing this suggestion or editing the file yourself)

**Comment by illfated** [comment]
#sign-off

**Pull request merged**